src/OpenRoad.cc

```diff
@@ -262,6 +262,12 @@ OpenRoad::init(Tcl_Interp *tcl_interp)
   // Import exported commands to global namespace.
   Tcl_Eval(tcl_interp, "sta::define_sta_cmds");
   Tcl_Eval(tcl_interp, "namespace import sta::*");
+
+  // Initialize tcl history
+  if (Tcl_Eval(tcl_interp, "history") == TCL_ERROR) {
+    // due to tcl history.tcl return error, remove error
```

Comment:
The comment should document the non-obvious reason for this special treatment: There appears to be a typo in the history.tcl file in some distributions, which is generating this error.

Reply:
I updated the comment as you suggested.